pkg/registry/registry.go

```diff
@@ -73,6 +73,26 @@ var (
 	globalRegistryByName map[uint32]map[string]*entities.InfoElement
 )
 
+func InitNewRegistry(customEnterpriseID uint32) error {
+	if globalRegistryByID == nil {
+		return fmt.Errorf("Please use LoadRegistry before registering custom registry")
+	}
+	globalRegistryByID[customEnterpriseID] = make(map[uint16]*entities.InfoElement)
+	globalRegistryByName[customEnterpriseID] = make(map[string]*entities.InfoElement)
+	return nil
+}
+
+func RegisterIE(ie entities.InfoElement, enterpriseID uint32) error {
```

Comment:
I'd maybe rename the function to something like `RegisterInfoElement` put `PutInfoElement`.